lib/web/join_tokens.go

```diff
@@ -392,9 +392,17 @@ func getJoinScript(ctx context.Context, settings scriptSettings, m nodeAPIGetter
 	// By default, it will use `stable/v<majorVersion>`, eg stable/v12
 	repoChannel := ""
 
-	// For Teleport Cloud installations, use the `stable/cloud` channel.
-	if settings.stableCloudChannelRepo {
+	// For Teleport Cloud installations with AutomaticUpgrades enabled, use the `stable/cloud` repo channel and get version from
```

Comment:
I would update this comment to explicitly mention that currently auto-upgrades in the install script are enabled _only_ for Cloud customers enrolled in auto-upgrades. Otherwise, it looks strange that we're checking generic "installUpdater" variable but override the channel to the cloud one.

Reply:
Updated to:
```

	// The install script will install the updater (teleport-ent-updater) for Cloud customers enrolled in Automatic Upgrades.
	// The repo channel used must be `stable/cloud` which has the available packages for the Cloud Customer's agents.
	// It pins the teleport version to the one specified by https://updates.releases.teleport.dev/v1/stable/cloud/version
	// This ensures the initial installed version is the same as the `teleport-ent-updater` would install.
```